[2019-02-17 22:01:08,693] 48721_worker_server INFO:Socket starts at 127.0.0.1:48721
[2019-02-17 22:01:08,693] 48721_worker_server INFO:Listening...
[2019-02-17 22:01:13,430] Client_Thread-1 INFO:Got connection from 127.0.0.1:34528
[2019-02-17 22:01:13,431] 48721_worker_server INFO:Listening...
[2019-02-17 22:01:13,431] Client_Thread-1 INFO:The connection's group name is admin
[2019-02-17 22:01:13,431] Client_Thread-1 INFO:Recieving the Python script file...
[2019-02-17 22:01:13,474] Client_Thread-1 INFO:Writing the Python script file to /home/zhiangz/Documents/48721_ass3_part1/rmt_worker/log/admin/Eplus-env-run43...
[2019-02-17 22:01:13,474] Client_Thread-1 INFO:Writing is finished
[2019-02-17 22:01:13,474] Client_Thread-1 INFO:Starting to run the script
[2019-02-17 22:01:13,476] Client_Thread-1 INFO:Finished task for 127.0.0.1:34528
[2019-02-17 22:01:14,026] Client_Thread-1 INFO:2019-02-17 22:01:14,026-xremote-INFO-2323220

[2019-02-17 22:01:14,528] Client_Thread-1 INFO:2019-02-17 22:01:14,528-xremote-INFO-2323221

[2019-02-17 22:01:15,030] Client_Thread-1 INFO:2019-02-17 22:01:15,030-xremote-INFO-2323222

[2019-02-17 22:01:15,532] Client_Thread-1 INFO:2019-02-17 22:01:15,532-xremote-INFO-2323223

[2019-02-17 22:01:16,034] Client_Thread-1 INFO:2019-02-17 22:01:16,034-xremote-INFO-2323224

[2019-02-17 22:01:16,536] Client_Thread-1 INFO:2019-02-17 22:01:16,535-xremote-INFO-2323225

[2019-02-17 22:01:17,037] Client_Thread-1 INFO:2019-02-17 22:01:17,037-xremote-INFO-2323226

[2019-02-17 22:01:17,538] Client_Thread-1 INFO:2019-02-17 22:01:17,538-xremote-INFO-2323227

[2019-02-17 22:01:18,039] Client_Thread-1 INFO:2019-02-17 22:01:18,039-xremote-INFO-2323228

[2019-02-17 22:01:18,547] Client_Thread-1 INFO:2019-02-17 22:01:18,541-xremote-INFO-2323229

[2019-02-17 22:01:18,547] Client_Thread-1 INFO:Zipping the results...
[2019-02-17 22:01:18,550] Client_Thread-1 INFO:Zipping is successful
[2019-02-17 22:01:18,553] Client_Thread-1 ERROR:Error is encountered when zipping the results, Traceback (most recent call last):
  File "/home/zhiangz/Documents/48721_ass3_part1/rmt_worker/server.py", line 135, in _python_process_handler
    c.sendall(f_line);
BrokenPipeError: [Errno 32] Broken pipe

